config_example.sh

```diff
@@ -197,6 +197,13 @@ set -x
 #export WORKER_DISK=20
 #export WORKER_VCPU=4
 
+# Add extradisks to VMs
+# export VM_EXTRADISKS=true
+
+# Configure how many extra disks to add to VMs. Takes a string of disk
+# names delimited by spaces. Example "vdb vde"
+# export VM_EXTRADISKS_LIST="vdb vde"
```

Comment:
nit, this example looks odd, wouldn't you generally expect contiguous device names, e.g `vdb vdc`?

Reply:
true, that will be better for an example. I sometimes flip "b" and "d" when I read these things 😅